tests/playwright/src/sso-extension.spec.ts

```diff
@@ -0,0 +1,160 @@
+
+  test.runIf(extensionInstalled)(
+    'Uninstalled previous version of sso extension',
+    async () => {
+      await removeExtension(extensionName);
+    },
+    60000,
+  );
+
+  test('Extension can be installed using OCI image', async () => {
+    const settingsBar = await navBar.openSettings();
+    const settingsExtensionPage = await settingsBar.openTabPage(SettingsExtensionsPage);
+    await playExpect(settingsExtensionPage.heading).toBeVisible();
+    await settingsExtensionPage.installExtensionFromOCIImage(imageName);
+  }, 200000);
+
+  test('Extension record is added under installed extensions', async () => {
+    const settingsBar = await navBar.openSettings();
+    const settingsExtensionPage = await settingsBar.openTabPage(SettingsExtensionsPage);
+    const extension = settingsExtensionPage.getExtensionRowFromTable(extensionLabelName);
+    await playExpect(extension).toBeVisible();
+    await extension.scrollIntoViewIfNeeded();
+    const extensionDetails = extension.getByRole('cell', { name: 'Extension Details' });
+    const details = extensionDetails.getByLabel('Connection Status Label');
+    await playExpect(details).toHaveText('RUNNING');
+  });
+
+  test('Extension appears under Settings bar extensions', async () => {
+    const settingsBar = await navBar.openSettings();
+    const extensions = await settingsBar.getCurrentExtensions();
+    await playExpect.poll(async () => await extensionExists(extensions, extensionName), { timeout: 30000 }).toBeTruthy();
+  });
+
+  test('SSO provider is available in Authentication Page', async (context) => {
+    console.log(`Running Test: ${context.task.name}`);
+    console.log(`Opening settings`);
+    const settingsBar = await navBar.openSettings();
+    const authPage = await settingsBar.openTabPage(AuthenticationPage);
+    await playExpect(authPage.heading).toHaveText('Authentication');
+    const provider = authPage.getProvider(authProviderName);
+    await playExpect(provider.getByLabel('Provider Information').getByLabel('Provider Name')).toHaveText(authProviderName);
+    await playExpect(provider.getByLabel('Provider Information').getByLabel('Provider Status')).toHaveText('Logged out');
+    await playExpect(provider.getByLabel('Provider Actions').getByRole('button')).toContainText('Sign in');
+  });
+
+  describe('Red Hat Authentication extension handling', async () => {
+    test('Extension can be disabled', async () => {
+      const settingsBar = await navBar.openSettings();
+      const settingsExtensionPage = await settingsBar.openTabPage(SettingsExtensionsPage);
+      const extension = settingsExtensionPage.getExtensionRowFromTable(extensionLabelName);
+      await extension.scrollIntoViewIfNeeded();
+      await settingsExtensionPage.getExtensionStopButton(extension).click();
+      const extensionDetails = extension.getByRole('cell', { name: 'Extension Details' });
+      await playExpect(extensionDetails.getByLabel('Connection Status Label')).toHaveText('OFF');
```

Comment:
Same as a previous point, this property is already available from the POM model, exposed from the extension page.